src/inventory.cc

```diff
@@ -271,7 +272,7 @@ static void _container_exit(int keyCode, int inventoryWindowType);
 static int _drop_into_container(Object* a1, Object* a2, int a3, Object** a4, int quantity);
 static int _drop_ammo_into_weapon(Object* weapon, Object* ammo, Object** a3, int quantity, int keyCode);
 static void _draw_amount(int value, int inventoryWindowType);
-static int inventoryQuantitySelect(int inventoryWindowType, Object* item, int a3);
+static int inventoryQuantitySelect(int inventoryWindowType, Object* item, int max, int defaultValue=1);
```

Comment:
```suggestion
static int inventoryQuantitySelect(int inventoryWindowType, Object* item, int max, int defaultValue = 1);
```
Eliminates Clang warning.